app/src/test/java/edu/usf/cutr/otp/demo/chicago/BikeRentalApiUnitTest.kt

```diff
@@ -0,0 +1,62 @@
+package edu.usf.cutr.otp.demo.chicago
```

Comment:
These tests should be moved to the `/library/src/androidTests` folder if they require Android (but it looks like they don't?), and if not they should be in the `/library/src/jvmTest` folder. Currently, if we do a command like `./gradlew :library:build` these tests won't run, because they aren't in the library project.

Reply:
Thanks for letting me know. The android unit tests run on the CI and if I copy these to the test folders, it should still work with the JVM.